src/builders/thermodynamic.jl

```diff
@@ -0,0 +1,252 @@
+"""
+$(TYPEDSIGNATURES)
+
+Build a max min driving force analysis model. See the docstring of
+[`max_min_driving_force_analysis`](@ref) for details about the arguments.
+"""
+function build_max_min_driving_force_model(
+    model::A.AbstractFBCModel,
+    reaction_standard_gibbs_free_energies::Dict{String,Float64};
+    reference_flux = Dict{String,Float64}(),
```

Comment:
the default doesn't really make sense here since it's gonna throw DomainError on all uses

Reply:
you mean for the `reference_flux`? If you supply all the dG0s, then it will be fine